src/pages/mobile/NamesNew.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <MobilePage
+    :header-fill="activeColor"
+    class="names-new"
+    right-button-icon-name="close"
+    @right-button-click="$router.back()"
```

Comment:
This may drop user out of the app if the user starts with this route. I propose to navigate the user explicitly to name list.